src/query/settings/src/lib.rs

```diff
@@ -269,6 +269,26 @@ impl Settings {
                 desc: "Timezone, default value: \"UTC\".",
                 possible_values: None,
             },
+            SettingValue {
+                default_value: UserSettingValue::String("row".to_owned()),
+                user_setting: UserSetting::create(
+                    "row_tag",
+                    UserSettingValue::String("row".to_owned()),
+                ),
+                level: ScopeLevel::Session,
+                desc: "In xml format, this field is represented as a row tag, e.g. <row>...</row>.",
+                possible_values: None,
+            },
+            SettingValue {
+                default_value: UserSettingValue::String("data".to_owned()),
+                user_setting: UserSetting::create(
+                    "rowset_tag",
+                    UserSettingValue::String("data".to_owned()),
+                ),
+                level: ScopeLevel::Session,
+                desc: "In xml format, this fields is represented as a row set tag, e.g.  <list> <row>...</row>, <row>...</row> </list>.",
```

Comment:
example with default value `data`?

Reply:
not clear where `rowset_tag` is used from the `desc`

Reply:
```
<?xml version="1.0"?>

<!-- rowset tag -->
<data>
  <!-- row tag -->
  <row title="Transaction Processing" author="Jim Gray" date="1992"/>
</data>

<data>
  <row>
  <title>Readings in Database Systems</title>
  <author>Michael Stonebraker</author>
  <date>2004</date>
  </row>
</data>

<data>
  <row>
  <field name="title">aaaa</field>
  <field name="author">bbbb</field>
  <field name="date">2022</field>
  </row>
</data>
```

Reply:
seems mysql does not use `rowset tag`, only `row tag`
 https://dev.mysql.com/doc/refman/8.0/en/load-xml.html.

can you explain some explanation why we need it?
if not highly necessary, I prefer fewer options.

Reply:
yes, mysql is use <list> tag to wrap the row.  this forces the user to use <list>,which is unreadable if the user uses <data> tag to wrap the row. 
```
<list>
<row_tag>
..
</row_tag>
</list>
```

another problem is that the user's xml might be, then have to specify the rowset tag.
```
<list>
<row_tag>
..
</row_tag>
</list>

<list>
<row_tag>
..
</row_tag>
</list>
```
🤔 but if we need to simplify options, we can specify it explicity in the dosc.

Reply:
i do this to make the user more flexible, maybe im just overthinking it.

Reply:
`rowset tag` is `list`?

Reply:
do you mean `row_tag` should be under `rowset tag`?

Mysql seems to only match the tag no matter which level is in.

e.g.  `address` is under `person` in the example (search  ` ROWS IDENTIFIED BY '<address>'` in https://dev.mysql.com/doc/refman/8.0/en/load-xml.html)

Reply:
i re-checked my code an recalled why i used rowset_tag eariler, its stupid. 😭
i just want to fit the following format:
```
<data>
  <row>
  <title>Readings in Database Systems</title>
  <author>Michael Stonebraker</author>
  <date>2004</date>
  </row>
</data>
```
when the attrs of tag is empty, only has three possibilities. <rowset_tag>,<row_tag>,<column_tag>.
i did a really stupid thing where i added a <rowset_tag> options to partition and get the <column_tag>.

🧐 i can use a temp variable such as has_row_start to record the start of the row and remove the <rowset_tag>